samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -691,6 +691,19 @@ To test this feature, complete the following steps:
       #. Start the ringing action on your device by tapping the :guilabel:`Play sound` button.
       #. Observe that **LED 1** is lit to confirm that the Android device is able to connect to your development kit after a clock synchronization.
 
+Disablement instructions
+------------------------
+
+To disable the locator tag device, complete the following steps:
+
+.. note::
+   Disablement instructions are used in Android and iOS to provide information on how to disable unknown trackers from each vendor.
+   These disablement instructions section is associated with the default debug device model registered for this sample (see the :ref:`fast_pair_locator_tag_google_device_model` section) and is used for demonstration purposes.
```

Comment:
What you mean by `These disablement instructions section is associated with the default debug device model registered for this sample`?